src/prefect/tasks/mysql/mysql.py

```diff
@@ -136,16 +138,18 @@ def __init__(
         self.query = query
         self.commit = commit
         self.charset = charset
+        self.cursor_type = cursor_type
         super().__init__(**kwargs)
 
-    @defaults_from_attrs("fetch", "fetch_count", "query", "commit", "charset")
+    @defaults_from_attrs("fetch", "fetch_count", "query", "commit", "charset", "cursor_type")
     def run(
         self,
         query: str,
         fetch: str = "one",
         fetch_count: int = 10,
         commit: bool = False,
         charset: str = "utf8mb4",
+        cursor_type: Union[str, Callable] = "cursor",
```

Comment:
I would be a little more specific about the type here
```suggestion
        cursor_type: Union[str, pymysql.cursors.Cursor] = pymysql.cursors.Cursor,
```
All of the other cursors inherit from the `Cursor` class and we can specify that this is what we are looking for

I'd also prefer to default to the real class type because it's easier for a user to know what we mean by the default value and it's already imported

Reply:
See Jim's comment at https://github.com/PrefectHQ/prefect/pull/3574#discussion_r513530489